### ISSUE TITLE
Report a problem on this page component: navigation back

### ISSUE THREAD

**Issue by ciclerou** [wet-boew/GCWeb#1208]
For the "report a problem" component, is it possible to add a Back button so that when used, it will be able to navigate back to previous page.   Current this fails WCAG 3.2.3 Consistent Navigation - Level AA.

**Comment by rlambert27** [comment]
Can you elaborate on the failure and which technique or common failure is achieved.

The component in AEM (https://www.canada.ca/en.html) is a form that is submitted directly from the page and there is no navigation.

Unless you are talking about the button being a link to another page. I don't see how that would be a failure of 3.2.3 though. The button provides the same functionality as a "Contact us" link and normally those don't have a mechanism to return to the previous page unless you use the browser back button.

**Comment by ciclerou** [comment]
Oh wow, yes I can see that it just opens up from that Canada.ca site and does not redirect, but for applications, when you click on the link in the footer, it redirects to that web page  you mention and you can only get back to the application website  if you click on the browser back button twice after you have submit a comment.  I will ask George to comment

**Comment by rlambert27** [comment]
Yes and applications have the possibility of hiding this button if they don't want to have it visible.

Also i'm not sure how different this button is to a contact us link for example. If you're in an application and you have issues and you hit contact us you don't normally have a button to return where you were.

Also 3.2.3 talks about the consistent navigation patterns that are presented on multiple web pages. Are navigation mechanisms that are repeated on multiple web pages placed in the same relative order.

As far I know the button to report a problem is always in the same location and is also always identified in the same way. Not having a mechanism to come back to the previous page would not be a failure of 3.2.3.

**Comment by ciclerou** [comment]
Sorry Roch, tester used wrong failure #, but it is still an issue.  See QA tester comments below 

- navigation back from confirmation page is required in the form of Back button located after confirmation message.
- Having breadcrumbs on the top is not a very good way to provide navigation to a previous page because breadcrumbs located above confirmation message.

**Comment by kodecount** [comment]
Just random thought. But this would be a issue with the design on that application and not WET.

**Comment by kodecount** [comment]
Just random thought, but wouldn't this be a issue with the design on that application and not GCWeb (or WET).

To elaborate what I'm getting at the issue that would arise with WET adding a back button is it would reply on the URL because it would have no knowledge of the previous page the user came from. Great example of this is if you accessed this link: http://wet-boew.github.io/themes-dist/GCWeb/report-problem-en.html From this GitHub comment it would have to rely on this part of URL http://wet-boew.github.io/themes-dist/GCWeb/ to be valid which it isn't.

The only other possible way is to have a button that goes back through the browsers history, but this could be miss leading. Depending on how or where the user accessed the link.

The issue with anything above is it all relys on JavaScript once it is turned of the back button would exist. If you wanted one that always worked you it have to be code on server-side and not WET because of these issue.

**Comment by jkshapiro** [comment]
Ironically, the page https://design.canada.ca/common-design-patterns/report-problem.html, which is about how to implement a "Report a problem" widget, has a "Report a problem" widget with just this problem. The form should open in-page, not on a separate page.
